### PR TITLE
DEV-3593: Rename toptier/submission cgac_code to toptier_code

### PR DESCRIPTION
**Pull request by kbard**
**Description:**

Renamed ToptierAgency and SubmissionAttributes `cgac_code` fields to `toptier_code` since they represent toptier agencies which are represented by either CGACs or FRECs.

**Matching front end PR:** [#1324](https://github.com/fedspendingtransparency/usaspending-website/pull/1324)

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] Didn't see any API documentation requiring updates
3. [x] Necessary PR reviewers:
    - [x] Backend
4. [x] Matview impact assessment completed
5. [x] Frontend impact assessment completed
6. [x] Data validation completed
7. [x] No Operations ticket required aside from a standard migration which is already slated to occur
8. [x] Jira Ticket [DEV-3593](https://federal-spending-transparency.atlassian.net/browse/DEV-3593):
    - [x] Link to this Pull-Request

### CODE REVIEW

File: usaspending_api/references/v1/serializers.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class ToptierAgencySerializer(LimitableSerializer):
     class Meta:
         model = ToptierAgency
         fields = "__all__"
-        default_fields = ["cgac_code", "name", "abbreviation"]
+        default_fields = ["toptier_code", "name", "abbreviation"]
```
Comment (tony-sappe) [inline usaspending_api/references/v1/serializers.py]: Does the frontend use any endpoints which use this serializer?

Comment (kbard) [reply usaspending_api/references/v1/serializers.py]: You mean not addressed by https://github.com/fedspendingtransparency/usaspending-website/pull/1324?

Comment (kbard) [reply usaspending_api/references/v1/serializers.py]: I fixed all references to `cgac_code` that I found in the front end.  I didn't check to see if/which endpoints were actually being used, if that's what you're asking.

Comment (tony-sappe) [reply usaspending_api/references/v1/serializers.py]: ah, OK. I wasn't aware of the frontend PR

**Pull request merged**